.github/workflows/check_nims.yml

```diff
@@ -76,7 +80,7 @@ jobs:
           mkdir -p docs   
           poetry run sphinx-build _docs_source docs -b html -W --keep-going
       - name: Revert docs
-        run:  git clean -dfx docs/ && git restore docs/
+        run:  git clean -dfx ../../docs/ && git restore ../../docs/
```

Comment:
Are you sure this is dirtying /docs and not /packages/service/docs?

Reply:
When I run it locally, it is putting them in /packages/service/docs. But when I left it as 'docs/', it gave an error that there was no docs/ directory. Maybe because there's not originally any source in GitHub at /packages/service/docs? I'll try again with docs/. Perhaps I need to make it just call 'rm -rf docs' again instead?

Reply:
Yeah. Here's the error. I guess I'll try 'rm -rf docs'
![image](https://github.com/ni/measurement-plugin-python/assets/38357562/93d14478-d61c-49c2-953f-7621eb8d96d8)

Reply:
That worked.